pylti/flask.py

```diff
@@ -27,6 +27,11 @@
 log = logging.getLogger('pylti.flask')  # pylint: disable=invalid-name
 
 
+def default_error(exception=None):
```

Comment:
nit: Though the testing doesn't use pylint, I see some pylint comments around. Should a pylint comment be added here for unused-arguments?